README.md

```diff
@@ -150,6 +150,24 @@ let scaleString = scale.stringValue
 let scaleInt = scale.integerValue
 ```
 
+Always use Swift equivalent of an Objective-C function whenever possible. For example when manipulating a CGRect variable:
+
+**Preferred:**
+```swift
+let rect = CGRect()
+let width = rect.width
+let height = rect.height
+```
+
+**Not Preferred:**
```

Comment:
Other `Not preferred` in the guide don't use capital 'P'.. Please fix then you can merge